azure/packages/test/scenario-runner/package.json

```diff
@@ -31,14 +31,14 @@
 		"start": "node ./dist/orchestratorRunner.js --config v1",
 		"start:server": "node ./dist/server.js",
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"test:realsvc": "npm run test:realsvc:tinylicious",
+		"test:realsvc": "echo Test disabled on LTS",
```

Comment:
Note for reviewers: I disabled this because:
1) This actually wasn't running on existing azure pipeline runs (one example: https://dev.azure.com/fluidframework/internal/_build/results?buildId=254938&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=8ee6b0c5-48a4-51bb-d1f6-a7ad1e52f3d2 )
2) The test steps weren't set up right - `test:realsvc:tinylicious` tries to invoke `start:tinylicious:test`, but that script doesn't actually exist, so it fails. I did some best effort work to enable this, but when I got the tests running I found there were too many failures for me to reasonably address in this PR and decided getting it working was out of scope